src/renderers/webxr/WebXRManager.js

```diff
@@ -338,6 +351,9 @@ class WebXRManager extends EventDispatcher {
 				customReferenceSpace = null;
 				referenceSpace = await session.requestReferenceSpace( referenceSpaceType );
 
+				currentPixelRatio = renderer.getPixelRatio();
+				renderer.getSize( currentSize );
```

Comment:
When I understand the code correctly, the idea is to save the original pixel ratio and dimension of the renderer and restore these values in `onSessionEnd()`.

If so, I think both lines should be positioned further up in the method before the values are overwritten in the previous `if/else` statement.

Reply:
It's a good thing this was caught.